packages/salesforcedx-vscode-core/package.nls.ja.json

```diff
@@ -3,12 +3,12 @@
   "analytics_generate_template_text": "SFDX: サンプルの Analytics テンプレートを作成",
   "apex_generate_class_text": "SFDX: Apex クラスを作成",
   "apex_generate_trigger_text": "SFDX: Apex トリガを作成",
-  "cancel_sfdx_command_text": "SFDX: アクティブなコマンドをキャンセル",
+  "cancel_sf_command_text": "SFDX: アクティブなコマンドをキャンセル",
   "config_list_text": "SFDX: すべての設定変数を一覧表示",
   "config_set_org_text": "SFDX: デフォルトの組織を設定",
   "conflict_detect_open_file": "ファイルを開く",
   "conflict_detect_resolve_view": "Org Differences",
-  "conflict_detection_enabled_description": "ソース追跡されない組織に対して、メタデータ操作 (マニフェストを使用した force:source:deploy または force:source:retrieve コマンド) の競合を検知するかどうかを指定します。",
+  "conflict_detection_enabled_description": "ソース追跡されない組織に対して、メタデータ操作 (マニフェストを使用した project:deploy:start または force:source:retrieve コマンド) の競合を検知するかどうかを指定します。",
```

Comment:
Japanese translation does not match English text - replace with English text to be re-translated.